build.gradle

```diff
@@ -25,20 +25,18 @@ import org.ajoberstar.gradle.git.tasks.*
 buildscript {
     repositories {
         mavenCentral()
-        maven {
-            url 'http://dl.bintray.com/jfrog/jfrog-jars'
-        }
+        jcenter()
     }
 
     dependencies {
         // Artifactory plugin
-        classpath(group: 'org.jfrog.buildinfo', name: 'build-info-extractor-gradle', version: '2.2.3')
+        classpath(group: 'org.jfrog.buildinfo', name: 'build-info-extractor-gradle', version: '4.0.0')
```

Comment:
Good point! I still need to update some of the other projects and do a Gestalt and TeraMath release :(

So far just the engine job is updated plus the engine PR and the module template